exercises/affine-cipher/description.md

```diff
@@ -10,61 +10,54 @@ its new numeric value. Although all mono-alphabetic ciphers are weak,
 the affine cipher is much stronger than the atbash cipher,
 because it has many more keys.
 
+## Encryption
+
 The encryption function is:
 
-  `E(x) = (ax + b) mod m`
-  -  where `x` is the letter's index from 0 - length of alphabet - 1
-  -  `m` is the length of the alphabet. For the roman alphabet `m == 26`.
-  -  and `a` and `b` make the key
+  `E(x) = (ai + b) mod m`
+  -  where `i` is the letter's index from `0` to the length of the alphabet - 1
+  -  `m` is the length of the alphabet. For the roman alphabet `m` is `26`.
+  -  `a` and `b` are integers which make the encryption key
+
+Values `a` and `m` must be *coprime* (or, *relatively prime*) for automatic decryption to succeed,
+ie. they have number `1` as their only common factor (more information about coprime numbers
+can be found [here](https://en.wikipedia.org/wiki/Coprime_integers)). In case `a` is not coprime to `m` your
+program should return status `1` and `"Error: a and m must be coprime."`. Otherwise it should encrypt or
```

Comment:
**optional**, since it was already this way before this PR: "return status `1`" is too specific; not all languages should signify errors this way. My suggestion will be "It is an error if `a` is not coprime to `m`."

Reply:
Again, I agree and have changed the text.